commit 795f18c3b79f5a90cafc9bebb378ec1881a2f08f
Author: Michael Terry <mike@mterry.name>
Date:   Wed Jan 5 10:52:11 2022 -0500

    main: adjust default size to be larger
    
    I had based my previous default size on what worked in X11, but
    there is a bug right now with mutter and X11 that causes those
    default sizes to result in much larger sizes.
    
    Instead, this should work on wayland and non-mutter WMs by
    default, but will unfortunately leave mutter & X11 users with
    large windows. But that's better than small windows.
    
    https://gitlab.gnome.org/World/deja-dup/-/issues/259

diff --git a/data/ui/MainWindow.ui b/data/ui/MainWindow.ui
index 85ca87da..356517ed 100644
--- a/data/ui/MainWindow.ui
+++ b/data/ui/MainWindow.ui
@@ -6,7 +6,8 @@ SPDX-FileCopyrightText: Michael Terry
 
 <interface>
   <template class="MainWindow" parent="AdwApplicationWindow">
-    <property name="default-height">400</property>
+    <property name="default-width">600</property>
+    <property name="default-height">500</property>
     <property name="title" translatable="yes">Backups</property>
     <property name="content">
       <object class="GtkBox">
diff --git a/data/ui/PreferencesWindow.ui b/data/ui/PreferencesWindow.ui
index 63e89331..7b1b90b3 100644
--- a/data/ui/PreferencesWindow.ui
+++ b/data/ui/PreferencesWindow.ui
@@ -7,8 +7,8 @@ SPDX-FileCopyrightText: Michael Terry
 <interface>
   <template class="PreferencesWindow" parent="AdwPreferencesWindow">
     <property name="modal">1</property>
-    <property name="default_width">400</property>
-    <property name="default_height">400</property>
+    <property name="default-width">500</property>
+    <property name="default-height">500</property>
     <property name="destroy_with_parent">1</property>
     <property name="search_enabled">0</property>
     <child>
